app-editors/ghostwriter/ghostwriter-1.8.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
+HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+BDEPEND="dev-qt/linguist-tools:5"
+
+RDEPEND="
+	app-text/hunspell
+	dev-qt/qtwebchannel:5
+	dev-qt/qtconcurrent:5
```

Comment:
`dev-qt/qtconcurrent:5` is still not being linked with ;)

Reply:
Seems that it's not being linked because of the -Wl,--as-needed flag. But is it a problem? The binary is explicitly linking with Qt5Concurrent. I'm not sure if I have understood you correctly.

Reply:
It means that being in `DEPEND` only was right.

Reply:
I think i'm a bit lost. If some user have some custom LDFLAGS or something and the compiler doesn't remove the dynamic linking with Qt5concucrrent, wouldn't cause problems to the user?

Reply:
we can't save users from themselves